core/core_bind.cpp

```diff
@@ -2184,6 +2185,82 @@ void ClassDB::_bind_methods() {
 
 } // namespace special
 
+////// _ScriptServer //////
+
+bool ScriptServer::_set(const StringName &p_name, const Variant &p_value) {
+	return false;
+}
+
+bool ScriptServer::_get(const StringName &p_name, Variant &r_ret) const {
+	if (::ScriptServer::is_global_class(p_name)) {
+		r_ret = ::ResourceLoader::load(::ScriptServer::get_global_class_path(p_name), "Script");
+		return true;
+	}
+	return false;
+}
+
+void ScriptServer::_get_property_list(List<PropertyInfo> *p_list) const {
+	ERR_FAIL_COND(!p_list);
+	List<StringName> names;
+	::ScriptServer::get_global_class_list(&names);
+	for (List<StringName>::Element *E = names.front(); E; E = E->next()) {
+		StringName n = E->get();
+		String class_name = String(n).get_file().get_extension();
```

Comment:
You're right! This code is total bogus. 😆

Reply:
Actually, if VSCode doesn't fail me, that method is never called. 👀

Reply:
Oh, vscode IS failing. Its super method is called of course.

Reply:
@willnationsdev Actually, aside of _bind_methods, apparently no method of ScriptServer is called. I set break points there but there not called when running tests or Godot itself. Any ideas here?

Reply:
`core_bind` contains the script bindings for various core classes, so it's natural that the methods would not be included in the codebase anywhere. They are only exposed here so that there is an actual `Object` with a `_bind_methods` function that binds the method implementations to scripting. This is how, e.g. GDScript is able to do `ScriptServer.get_global_class_name(<path>)`, etc.

As for why that line is doing the formatting checks that way, I honestly kind of forget exactly why. Technically speaking, it's a way to forcibly get the "name" that follows any given `/` or `.`; however, I can't recall if those methods would just return the original string if they don't find a `/` or `.`, or if they return an empty string. If the latter, then those may need to be removed. And if the former, I can't really recall why they would've been there to begin with since, yes, iirc, the global class names list should consist entirely of class names rather than filepaths.

Reply:
As it works as it is currently, I'd like to keep this in unless a core dev complains about it as well.